_includes/head.html

```diff
@@ -24,30 +23,27 @@
 };
 </script>
 
-{% include_cached vwo.html %}
```

Comment:
We can remove this from the `_includes` folder then?

Reply:
Yes we can going to remove it